src/tokens2.jl

```diff
@@ -66,7 +66,15 @@ end
          ii[2].address == 1?                       2 :
          ii[2].address == 2?                       3 : 1
 
-
+"""
+    compare(m::SAContainer, s::IntSemiToken, t::IntSemiToken)
+
+Here, `s`  and `t`  are semitokens  for the  same container  `m`; this
```

Comment:
`Here, `s`  and `t`  are semitokens  for the  same container  `m`;` should come at the end (and "here" isn't useful).